pipelines/_Multiband.yaml

```diff
@@ -2,7 +2,7 @@ description: Multiband
 tasks:
   detection: lsst.pipe.tasks.multiBand.DetectCoaddSourcesTask
   mergeDetections: lsst.pipe.tasks.mergeDetections.MergeDetectionsTask
-  deblend: lsst.pipe.tasks.deblendCoaddSourcesPipeline.DeblendCoaddSourcesSingleTask
+  deblend: lsst.pipe.tasks.deblendCoaddSourcesPipeline.DeblendCoaddSourcesMultiTask
```

Comment:
I would like to see these pipelines be more documented. A comment here to the effect of "Use Scarlet for deblending following RFC-745" or something would be useful. `DeblendCoaddSourcesMultiTask` itself doesn't even have a docstring, so the relationship between this change and scarlet is unclear unless one already knows that "DeblendMulti" means "scarlet".